### DISCUSSION TITLE
[공지] 백준 학교등록 하는방법

### DISCUSSION THREAD

**Discussion by ghdcksgml1** [ghdcksgml1/Algorithm_Study#94] [공지사항]
### 백준 학교등록 하는 방법 입니다!

<br/>

# 1. 백준을 로그인 한뒤, 오른쪽 상단의 설정 버튼을 클릭한다.
<img width="1175" alt="스크린샷 2022-02-28 오후 9 02 00" src="https://user-images.githubusercontent.com/79779676/155980365-5ef996e0-2da1-4726-922a-21cd8c54c60c.png">

<br/>

# 2. 학교/회사 정보를 클릭한다.

<img width="1190" alt="스크린샷 2022-02-28 오후 9 02 37" src="https://user-images.githubusercontent.com/79779676/155980380-2fc48e73-a911-41fb-bf6d-34d5aa868db4.png">

# 3. 학교 이메일로 추가한다.

<img width="1175" alt="스크린샷 2022-02-28 오후 9 03 15" src="https://user-images.githubusercontent.com/79779676/155980383-ecb5f786-d7ac-4f94-9c52-4716bb8b00bb.png">

---

## 인증메일이 날아오지 않는 경우.

학교 이메일을 입력해도 인증메일이 날라오지 않는 경우가 있는데

이 경우에는 문의하기 버튼을 클릭해서 추가해달라고 문의메일을 넣으면 빠르게 추가해줍니다!

---

### 학교 인증이 완료되면, solved.ac에 다음과 같이 학교가 추가됩니다.

<img width="1248" alt="스크린샷 2022-02-28 오후 9 04 20" src="https://user-images.githubusercontent.com/79779676/155980385-dc199c35-8c04-4fa6-83d5-86241114410b.png">

### 학교 랭킹도 확인할 수 있습니다.

<img width="1552" alt="스크린샷 2022-02-28 오후 9 00 02" src="https://user-images.githubusercontent.com/79779676/155980839-01e207f4-e9a5-4be5-b68c-be2801d82cff.png">

**Comment by jusung-c** [comment]
오 105등이다 
근데 티어 기준이 아니라 맞은문제 수 기준인가?
판수로 밀어붙여야지

**Reply by ghdcksgml1** [reply]
백준에서는 맞은 수 기준인데, 솔브닥에서는 티어기준!
